scripts/lustre

```diff
@@ -70,8 +70,8 @@ lustre_monitor() {
         if [ $mntdev = $OCF_RESKEY_dataset ] ; then
             ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
             return $OCF_SUCCESS
+        else ocf_log debug "$OCF_RESKEY_dataset does not match $mntdev"
```

Comment:
Cameron, I believe this will increase the number of debug messages for servers which mount more than one target (for example, zinc1).  Is that what you intended?

The previous version emitted a message outside the loop.  That meant it was reached only if all the mntdev paths were checked and none matched (because if one matched, we hit the "return" and do not execute the rest of the loop or anything after it in function lustre_monitor().

Reply:
My mistake, I misread this.  Ignore my earlier comment.